src/styles/blocks/card.scss

```diff
@@ -0,0 +1,132 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  width: 200px;
+  height: fit-content;
+  box-sizing: border-box;
+  margin: auto;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: all $animation-duration ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info-area {
+    margin: 40px 0 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    color: #060b35;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: left;
+    justify-content: left;
+    margin: 0 0 4px;
+    padding: 0;
+    transition: color $animation-duration ease;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0;
+    text-align: left;
+    justify-content: left;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__review-area {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    font-size: 10px;
+    font-weight: 400;
+    letter-spacing: 0;
+    text-align: right;
+    margin: auto;
+  }
+
+  &__price-area {
+    display: flex;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 0;
+    text-align: center;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: left;
+    display: inline;
+    margin: 0;
+    color: #616070;
+    transition: all $animation-duration ease;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: right;
+    display: inline;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    color: #fff;
+    text-decoration: none;
+    line-height: 40px;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    letter-spacing: 0;
+    text-align: center;
+    margin: 16px 0 0;
+    padding: 0;
+    transition: all $animation-duration ease;
+  }
+
+  &__button:hover {
```

Comment:
Rafael, this :hover could be done using nesting too
```
&__button{  
    &:hover{
    }
} 
```

Also, I'll be extra picky and point that on hover the button's text is slightly moving down, just a little. Can you guess why?

Reply:
Yes, I see is because I add borders in hover efect. Solution is to put border transparent before hover.